content/en-us/projects/assets/api.md

```diff
@@ -1,195 +1,283 @@
+    <tr>
+      <td>creatorType</td>
+      <td>int</td>
+      <td>`1` = User<br />`2` = Group</td>
+    </tr>
+    <tr>
+      <td>creatorTargetId</td>
+      <td>int64</td>
+      <td>User or Group Id to search for</td>
+    </tr>
+    <tr>
+      <td>minDuration</td>
+      <td>int64</td>
+      <td>Minimum value of the duration range for audio assets in seconds</td>
+    </tr>
+    <tr>
+      <td>maxDuration</td>
+      <td>int64</td>
+      <td>Maximum value of the duration range for audio assets in seconds</td>
+    </tr>
+    <tr>
+      <td>sortDirection</td>
+      <td>int</td>
+      <td>`0` = None<br />`1` = Ascending<br />`2` = Descending</td>
+    </tr>
+    <tr>
+      <td>artist</td>
+      <td>string</td>
+      <td>The name of the artist</td>
+    </tr>
+    <tr>
+      <td>album</td>
+      <td>string</td>
+      <td>The album you are looking for</td>
+    </tr>
+    <tr>
+      <td>audioTypes</td>
+      <td>array[int]</td>
+      <td>`0` = Music<br />`1` = SoundEffect</td>
+    </tr>
+    <tr>
+      <td>uiSortIntent</td>
+      <td>int</td>
+      <td>What sort order to rank the results by<br /><br />`1` = Relevance<br />`6` = Trending<br />`7` = AllTime<br />`8` = Top<br />`9` = Duration<br />`10` = DateCreated<br />`11` = DateModified<br />`12` = Creator<br />`13` = Name</td>
+    </tr>
+    <tr>
+      <td>includeOnlyVerifiedCreators</td>
+      <td>bool</td>
+      <td>A flag to include only results from verified creators. Verified creators are those that are ID or phone verified.</td>
+    </tr>
+    <tr>
+      <td>minPriceInCents</td>
+      <td>int64</td>
+      <td>Minimum cost in cents (only applicable to plugins)</td>
+    </tr>
+    <tr>
+      <td>maxPriceInCents</td>
+      <td>int64</td>
+      <td>Maximum cost in cents (only applicable to plugins)</td>
+    </tr>
+  </tbody>
 </table>
 
-The following URL will search for 10 items in the "Models" subcategory, sorted by most recently updated.
+For more detailed documentation regarding the toolbox-service, please refer to the updated documentation at creator store API.
 
-`https://search.roblox.com/catalog/json?Category=6&SortType=3&ResultsPerPage=10`
+This is a sample response from the endpoint:
 
-### Response Fields
+```json
+{
+  "totalResults": int,
+  "filteredKeyword": string,
+  "spellCheckerResult": {
+    "correctionState": int,
+    "correctedQuery": string,
+    "userQuery": string
+  },
+  "queryFacets": {
+    "appliedFacets": Array[string],
+    "availableFacets": Array[string]
+  },
+  "imageSearchStatus": int,
+  "previousPageCursor": string,
+  "nextPageCursor": string,
+  "data": [
+    {
+      "id": int,
+      "name": string,
+      "searchResultSource": string
+    }
+  ]
+}
+```
 
-API responses return in a JSON format. The response provides asset details with the following primary fields:
+To get back the asset metadata, you would go over the data array and extract the ids, call the GetItemDetails endpoint on toolbox-service like this:
```

Comment:
Is "GetItemDetails" a specific, proper API call or similar? Can we link to it elsewhere if so?

Reply:
Yeah, its a specific API. the example is given below, as mentioned. This API (toolbox-service) is a temporary workaround / API for creators until the creator store team builds a more robust API in Open Cloud, so these docs itself will be all that we give creators on how to use "search" in lieu of the catalog API deprecation.